templates/keycloak.service.j2

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Jboss Application Server
+After=network.target
+
+[Service]
+Type=idle
+Environment=JBOSS_HOME={{ keycloak_parent_install_dir }}/keycloak JBOSS_LOG_DIR={{ keycloak_log_dir }} "JAVA_OPTS=-Xms1024m -Xmx20480m -XX:MaxPermSize=768m"
```

Comment:
"JAVA_OPTS=-Xms1024m -Xmx20480m -XX:MaxPermSize=768m" This should be configurable at the least.

Its possible to run key cloak in less powerful servers and the default is 
 `-Xms64m -Xmx512m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m`

`XX:MaxPermSize` is removed af